tests/helpers/test_intent.py

```diff
@@ -68,6 +69,13 @@ async def test_async_match_states(hass):
         )
     )
 
+    # Test area alias
+    assert [state1] == list(
```

Comment:
Our convention is to put the expected value last, on the right side, of the assert statement.